webapp/app.py

```diff
@@ -11,6 +11,7 @@
 
 from database import db
 
+#This mail config is used for the creation of a user only | Try to remove it, but used in scripts/create_mock
```

Comment:
In general, you would like to define those kind of things once in the root module of your application.

Indeed, submodules as "users" can often be used in multiple projects, so you don't want to define configuration in those files. Moreover, you could need send emails in other submodules (e.g. in "lessons" to warn student about a deadline).